src/mo_frontend/typing.ml

```diff
@@ -148,10 +186,44 @@ let warn_lossy_bind_type env at bind t1 t2 =
 let _warn_in modes env at code fmt =
   ignore (diag_in type_warning modes env at code fmt)
 
+(* Unused declaration detection *)
+
+let emit_unused_warnings env =
+  let emit (id, region) = warn env region "M0194" "Unused declaration %s" id in
```

Comment:
```suggestion
  let emit (id, region) = warn env region "M0194" "unused identifier %s" id in
```
For consistency with other errors, probably best to lowercase the message. Also, the manual uses the terminology identifier/binding rather than variable/declaration but I don't feel that strongly about it.